### PR TITLE
Sg 1205 Accessibility/bug fixes

### PR DESCRIPTION
**Pull request by jacine**

### CODE REVIEW

File: web/modules/custom/sfgov_utilities/sfgov_utilities.module [hunk 1]
```diff
@@ -250,4 +251,31 @@ function _sfgov_utilities_build_map($address) {
...
+  }
+}
+
+/**
+ * Implements hook_form_alter().
+ */
+function sfgov_utilities_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+  // Attach #form_id to all form elements.
+  sfgov_utilities_attach_form_id($form, $form_id);
+}
```
Comment (arlina-espinoza) [inline web/modules/custom/sfgov_utilities/sfgov_utilities.module:281]: I'm curious as to why is this needed?

Comment (jacine) [reply web/modules/custom/sfgov_utilities/sfgov_utilities.module:281]: We need to make sure we are only targeting the checkboxes wrapper in the meeting filters form, and elements don't have that context, so we are passing that down to elements in here. The preprocess hook closer to the top is using this further up in the diff. 😀
